<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="MachineTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="DetMachineTable">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить 
 детерминированный 
 автомат</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VisB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Визуализировать</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="Viewer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>381</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Исходный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Детерминированный</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Строки:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rowsname">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Столбцы:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер(mxn):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Msize">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="colsname">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="strucB">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать автомат</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="DetViewer">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>381</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Последовательность:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rowsname_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="VisB_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>510</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>790</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>121</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
